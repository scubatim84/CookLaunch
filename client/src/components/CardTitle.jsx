import React from 'react';
import { Typography } from '@material-ui/core';

const CardTitle = (props) => (
  <Typography id="Title" component="h1" variant="h5">
    {props.title}
  </Typography>
);

export default CardTitle;
