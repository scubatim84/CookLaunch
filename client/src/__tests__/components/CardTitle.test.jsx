import React from 'react';
import {shallow} from 'enzyme';
import CardTitle from '../../components/CardTitle';

describe('CardTitle', () => {
  it('Renders component without crashing', () => {
    shallow(<CardTitle title='Test Title' />);
  });

  it('Renders title passed as prop to component', () => {
    const wrapper = shallow(<CardTitle title='Test Title' />);

    expect(wrapper.find('#Title').equals('Test Title'));
  });
});
