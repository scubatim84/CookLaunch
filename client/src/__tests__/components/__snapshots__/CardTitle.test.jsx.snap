import React from 'react';
import {shallow} from 'enzyme';
import CardTitle from '../../components/CardTitle';

describe('CardTitle', () => {
  it('Renders correctly', () => {
    const wrapper = shallow(<CardTitle />);

    expect(wrapper).toMatchSnapshot();
  });
});
